custom/icds/location_rationalization/tests/base.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+from corehq.util.workbook_json.excel import get_workbook
+from custom.icds.location_rationalization.parser import Parser
+
+
+class BaseTest(TestCase):
+    location_types = ['awc', 'supervisor', 'block', 'district', 'state']
+
+    def get_transitions(self):
+        filepath = "custom/icds/location_rationalization/tests/data/Location Rationalisation.xlsx"
```

Comment:
can you explain more, i don't understand why that would be better.
For debugging/troubleshooting i can view/edit the excel file, which should be as quick as modifying data in test? that method also eventually creates a workbook and then reads it back, why not just have an excel already.

Reply:
The main things I don't like are that they can't be code reviewed and that you can't make quick edits to them in github's web ui. What do you think?

Reply:
Oh damn, github makes me download the file to even view it.
i guess thats an improvement github needs to make :)
Should have at least let us view the file if not edit. 
I am just not fine with creating a file and then saving it and then reading it back each time this test is run and rather just ask reviewer to download and see it once or see on machine directly in case you have branch pulled. 
Both excel sheets are actually present here as well
[Good sheet](https://docs.google.com/spreadsheets/d/1QUe-eARxZ-ZISATp6kf6sYcfN_3KlmlC8J35GMirLXQ/edit#gid=412612614)
[Bad sheet](https://docs.google.com/spreadsheets/d/1QUe-eARxZ-ZISATp6kf6sYcfN_3KlmlC8J35GMirLXQ/edit#gid=1086128014)

Reply:
Okay, we can agree to disagree on this one.